### PR TITLE
feat: add _747.js

### PR DESCRIPTION
**Pull request by yaodehaung**

### CODE REVIEW

File: javascript/_747.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+  for( var i = 0 ;  i < nums.length; i++){
+      if(nums[i] > maxdata){
+          maxdata = nums[i];
+          maxIndex = i
+      }
+  }
+ 
+  for(var i = 0 ; i < nums.length; i++){
+
```
Comment (fishercoder1534) [inline javascript/_747.js:14]: Remove this empty line.

File: javascript/_747.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+          maxdata = nums[i];
+          maxIndex = i
+      }
+  }
+ 
+  for(var i = 0 ; i < nums.length; i++){
+      
+      if(i != maxIndex && nums[i] * 2 > maxdata){
+           return  -1
+
```
Comment (fishercoder1534) [inline javascript/_747.js:17]: Ditto.

File: javascript/_747.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
+var dominantIndex = function(nums) {
+      var maxdata = -1;
```
Comment (fishercoder1534) [inline javascript/_747.js:2]: Indentation line up please.

**Review by fishercoder1534 — CHANGES_REQUESTED** [review]

File: javascript/_747.js (hunk 3 above)
Comment (yaodehaung) [reply javascript/_747.js]: ok

**Comment by yaodehaung** [comment]
I solved the problem.

**Comment by WillShen7789** [comment]
good js

**Pull request closed without merging**